.pre-commit-config.yaml

```diff
@@ -54,7 +54,7 @@ repos:
     exclude: .github/labeler.yml|.pre-commit-search-and-replace.yaml
 
 - repo: https://github.com/mattlqx/pre-commit-search-and-replace
-  rev: v1.0.5
+  rev: v1.1.3
```

Comment:
I ran into some problems with v1.1.2 a while back (probably in the pre-commit autoupdate PRs), but v1.1.3 seems to be working. Yay!